package/patterns-yast.spec

```diff
@@ -90,27 +92,23 @@ Recommends:     yast2-samba-server
 # #388892
 Recommends:     yast2-vm
 # #542936
-Recommends:     yast2-packager-webpin
-Recommends:     yast2-auth-client
-Recommends:     yast2-fonts
-Recommends:     yast2-journal
 Recommends:     yast2-vpn
+# Recommend NTP at least until boo#936378 is fixed and YaST is not trying to configure a service that's not there
+Recommends:     ntp
 Suggests:       yast2-online-update-configuration
-Suggests:       yast2-product-creator
 Suggests:       autoyast2
 Suggests:       autoyast2-installation
 Suggests:       inst-source-utils
```

Comment:
The `inst-source-utils` was suggested for `yast2-product-creator` and can be dropped. The same applies for `createrepo`.